diff --git a/rustplus.js b/rustplus.js
index 44970966e617ed7e369e251690a714dbd830e5cb..94e22a2f7933f8dfb9c36560a0af939f82db8b03 100644
--- a/rustplus.js
+++ b/rustplus.js
@@ -44,7 +44,7 @@ class RustPlus extends EventEmitter {
     connect() {
 
         // load protobuf then connect
-        protobuf.load(path.resolve(__dirname, "rustplus.proto")).then((root) => {
+        protobuf.load(process.cwd() + "/app/rustplus.proto").then((root) => {
 
             // make sure existing connection is disconnected before connecting again.
             if(this.websocket){
