diff --git a/rustplus.js b/rustplus.js
index 44970966e617ed7e369e251690a714dbd830e5cb..ba587ec55477164a3b81b7425d3d7b63071d763c 100644
--- a/rustplus.js
+++ b/rustplus.js
@@ -44,7 +44,7 @@ class RustPlus extends EventEmitter {
     connect() {
 
         // load protobuf then connect
-        protobuf.load(path.resolve(__dirname, "rustplus.proto")).then((root) => {
+        protobuf.load(path.join(process.cwd(), "rustplus.proto")).then((root) => {
 
             // make sure existing connection is disconnected before connecting again.
             if(this.websocket){
@@ -151,6 +151,9 @@ class RustPlus extends EventEmitter {
             ...data, // merge in provided data for AppRequest
         });
 
+        // explicitly specify the player ID to prevent it from being converted to a number
+        request.playerId = this.playerId;
+
         // send AppRequest packet to rust server
         this.websocket.send(this.AppRequest.encode(request).finish());
 
