diff --git a/src/root.js b/src/root.js
index 2776f96e42994ce2e29aaed8b2ed9ff923b97e93..15255b3b28348880d2974c7b6953b173efd01076 100644
--- a/src/root.js
+++ b/src/root.js
@@ -1,6 +1,8 @@
 "use strict";
 module.exports = Root;
 
+const fs = require('fs');
+
 // extends Namespace
 var Namespace = require("./namespace");
 ((Root.prototype = Object.create(Namespace.prototype)).constructor = Root).className = "Root";
@@ -178,6 +180,9 @@ Root.prototype.load = function load(filename, options, callback) {
             }
             process(filename, source);
         } else {
+            console.log(filename + ' start loading');
+            fs.readFileSync(filename).toString("utf8");
+            console.log(filename + ' end loading');
             ++queued;
             self.fetch(filename, function(err, source) {
                 --queued;
